Lib/test/test_pathlib.py

```diff
@@ -2145,6 +2145,8 @@ def test_expanduser(self):
             otherhome = pwdent.pw_dir.rstrip('/')
             if othername != username and otherhome:
                 break
+        else:
+            self.skipTest('no other user with home != /')
```

Comment:
This will skip the whole test even if most checks are not related to othername.

I would set `othername = username` and `otherhome = userhome` or set `othername = None` and skip only checks for `p3`.